pkg/util_test.go

```diff
@@ -0,0 +1,163 @@
+/*
+ * Nuts crypto
+ * Copyright (C) 2019. Nuts community
```

Comment:
2020

Reply:
Shouldn't we fix it in all files? If so, I'd suggest we fix it in a seperate PR. (if not, I'll fix it here)